spec/requests/unwithdraw_spec.rb

```diff
@@ -65,9 +65,8 @@
       login_as(managing_editor)
 
       get unwithdraw_path(withdrawn_edition.document)
-      follow_redirect!
 
-      expect(response.body).to include(I18n.t!("documents.show.unwithdraw.title"))
+      expect(response.body).to include(withdrawn_edition.title_or_fallback)
```

Comment:
Switching to "unwithdraw.confirm.title" would be a closer like for like swap here.

Reply:
Thanks - yes, I thought about that but unlike the delete draft we already had a dedicated unwithdraw.yml.  So, rather than create an additional file - and as it seemed to fit the need - I opted to use that.  I can change if you'd prefer.

Reply:
Yeah I think we should try maintain the consistent approach. Is the existing file breaking this convention?

Reply:
Ok, got it - will change.

Reply:
Oops I think we chatted about this comment here: https://github.com/alphagov/content-publisher/pull/1611/files/78c5e74f4b8d48208c5bf117dad255d505e96ea0#diff-a94cb3ddb985768921b39dc10c93465c rather than this code change.

Reply:
Fixed